libs/libnetconf2/patches/001-cmake_not_updated.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt	2020-05-08 13:29:30.903800624 +0300
++++ b/CMakeLists.txt	2020-05-08 13:29:50.943582218 +0300
+@@ -153,7 +153,7 @@
+     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DNC_ENABLED_SSH")
+     message(STATUS "LibSSH version ${LibSSH_VERSION} found")
+ 
+-    if(LibSSH_VERSION VERSION_EQUAL 0.9.3 OR LibSSH_VERSION VERSION_EQUAL 0.9.4)
++    if(LibSSH_VERSION VERSION_EQUAL 0.9.x)
+         message(FATAL_ERROR "LibSSH ${LibSSH_VERSION} includes regression bugs and libnetconf2 will NOT work properly, try to use another version")
```

Comment:
Hmmmmmmmmmmmmm.....

Reply:
is just they didn't update the configure.ac despite the patch was merged in 0.9.4